metadata/service/gateway/single.yml

```diff
@@ -6,6 +6,7 @@ parameters:
   neutron:
     gateway:
       enabled: true
+      dns_domain: novalocal
```

Comment:
remove this line

Reply:
This is the fix to the bug I reported though. Removing this line the bug would still exist in default deployments

If you want to remove the default here, then you probably should also remove the default in nova
https://github.com/salt-formulas/salt-formula-nova/blob/master/metadata/service/control/single.yml#L16 , 
https://github.com/salt-formulas/salt-formula-nova/blob/master/metadata/service/control/container.yml#L18 , 
and 
https://github.com/salt-formulas/salt-formula-nova/blob/master/metadata/service/control/cluster.yml#L16

Would you rather fix the defaults in that package?

Reply:
Also in this package: 
https://github.com/salt-formulas/salt-formula-neutron/blob/master/metadata/service/control/single.yml#L13
https://github.com/salt-formulas/salt-formula-neutron/blob/master/metadata/service/control/container.yml#L14
https://github.com/salt-formulas/salt-formula-neutron/blob/master/metadata/service/control/cluster.yml#L11

Would you like me to remove those defaults as well to make it consistent?